src/main/java/de/tum/in/www1/artemis/repository/TextExerciseRepository.java

```diff
@@ -51,6 +47,18 @@ default TextExercise findByIdElseThrow(long exerciseId) {
         return findById(exerciseId).orElseThrow(() -> new EntityNotFoundException("Text Exercise", exerciseId));
     }
 
+    @Query("""
+            SELECT DISTINCT e FROM TextExercise e
+            LEFT JOIN FETCH e.gradingCriteria
+            WHERE e.id = :#{#exerciseId}
```

Comment:
Here is also the `:#{#exerciseId}` didn't you change the other to `:exerciseId`?

Reply:
I forgot this one 👍